### PR TITLE
added normalization and skull stripping

### PR DESCRIPTION
**Pull request by DAck29**

### CODE REVIEW

File: mialab/filtering/preprocessing.py [hunk 1]
```diff
@@ -77,10 +80,13 @@ def execute(self, image: sitk.Image, params: SkullStrippingParameters = None) ->
         """
         mask = params.img_mask  # the brain mask
 
+        if mask is None:
         # todo: remove the skull from the image by using the brain mask
-        warnings.warn('No skull-stripping implemented. Returning unprocessed image.')
+            warnings.warn('No skull-stripping implemented. Returning unprocessed image.')
```
Comment (alicjak1519) [inline mialab/filtering/preprocessing.py]:
```suggestion
            warnings.warn('No brain mask for skull-stripping provided. Returning unprocessed image.')
```

**Review by alicjak1519 — CHANGES_REQUESTED** [review]
lgtm, maybe we can just update the warning message, because the previous one is not true anymore thanks to this PR :)

File: mialab/filtering/preprocessing.py (hunk 1 above)
Comment (DAck29) [reply mialab/filtering/preprocessing.py]: Sounds good.

**Review by alicjak1519 — APPROVED** [review]
👌

**Pull request merged**